canonical-web-and-design/global-nav
# Update juju link to juju.is

## Done
Updated the link and removed juju login link.

## QA
- Run the demo
- See that the text matches the changes

## Screenshot
![image](https://user-images.githubusercontent.com/1413534/76333337-62a36980-62e9-11ea-9e42-0aa7a2b39bc2.png)
